res/controllers/Hercules DJ Console 4-Mx.midi.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version='1.0' encoding='utf-8'?>
 <MixxxControllerPreset mixxxVersion="2.0.0+" schemaVersion="1">
```

Comment:
We should do *something* (and dont currently I believe) -- whether it's warn vs refuse :shrug: